shared/src/main/scala/Elements.scala

```diff
@@ -448,6 +448,21 @@ object Elements:
       case (a: VFun, b) => MiscHelpers.collectUnique(a, b)
     }
 
+    val interleave: Dyad = addElem(
+      Dyad,
+      "I",
+      "Interleave",
+      List("interleave"),
+      "a: lst, b: lst -> Interleave a and b"
+    ) { case (a, b) =>
+      val temp = ListHelpers.interleave(
+        ListHelpers.makeIterable(a),
+        ListHelpers.makeIterable(b)
+      )
+      if a.isInstanceOf[String] && b.isInstanceOf[String] then temp.mkString("")
```

Comment:
```suggestion
      if a.isInstanceOf[String] && b.isInstanceOf[String] then temp.mkString
```
`mkString` uses `""` by default

Reply:
I blame copilot